### ISSUE TITLE
New Score Wizard not working with voice-over 4.2

### ISSUE THREAD

**Issue by musiccarsfan** [musescore/MuseScore#21079]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Bug description

When attempting to create a new score, the cancel and next buttons duplicate themselves, and I am unable to activate the next button by using the VO+space command. I am running Sonoma 14.2.1 on a 2022 m2 macbook pro using musescore 4.2. 

### Steps to reproduce

With voice-over turned on and Musescore open, press command+n for a new score. The templates list will come into focus. Try selecting the desired template by navigating to it using vo+arrow keys. To exit from the list of templates, press vo+shift+up arrow. Navigate using vo+arrows to the next button. Notice the cancel and next options appear to be duplicated. Try pressing vo+space or the return key on anyone of them. I have only gotten the button to activate once out of my 7 attempts. Please note vo = capslock and or command+option depending on voice-over settings. Thank you for your assistance with this crippling issue
<img width="1440" alt="Screenshot 2024-01-18 at 12 52 32 PM" src="https://github.com/musescore/MuseScore/assets/139388657/a2f35d00-19b1-4d9f-9412-ac982546e4f2">
.

### MuseScore Version

4.2

### Regression

Yes, this used to work in a previous version of MuseScore 4.x

### Operating system

Mac OS 14.2.1

**Comment by zacjansheski** [comment]
In 4.1.1, using the VO arrows still makes the done/next/cancel buttons not selectable

You will have better results if you use tab and arrow keys to get around and select


https://github.com/musescore/MuseScore/assets/69917893/99de5615-1ace-423c-ad74-d496553b7e3e

**Comment by shoogle** [comment]
As a Qt-based application, I don't think we have control over where the VoiceOver cursor goes, so definitely try using the Tab key, arrow keys, Spacebar and Enter all without the VO modifier.

**Comment by bkunda** [comment]
Closing at it appears to be a VO issue beyond our control.
